api/app-config.development.yaml

```diff
@@ -1,5 +1,11 @@
 auth:
-  enable: false
+  enable: true
+  oidc_providers:
+    - issuer_url: https://czi-prod.okta.com/oauth2/aus7jg49oduHuUSs25d7
+      client_id: test-test-test-authz
+    - issuer_url: https://token.actions.githubusercontent.com
```

Comment:
current example of allowing any github actions through that have the owner of "chanzuckerberg"